### PR TITLE
Feature al 491 rename endpoints and remove unused methods

### PR DESCRIPTION
**Pull request by dafinkagerdjikova**

### CODE REVIEW

File: src/Lykke.AlgoStore.Api/Controllers/AlgoController.cs [hunk 1]
```diff
@@ -59,5 +66,190 @@ public AlgoController(IAlgoStoreClientDataService clientDataService)
...
+
+            return Ok(response);
+
+        }
+
+        [HttpGet("algoMetadata")]
+        [SwaggerOperation("GetAlgoMetadata")]
+        [ProducesResponseType(typeof(AlgoDataInformationModel), (int)HttpStatusCode.OK)]
+        [ProducesResponseType(typeof(BaseErrorResponse), (int)HttpStatusCode.InternalServerError)]
+        [ProducesResponseType((int)HttpStatusCode.NotFound)]
```
Comment (iliyan-ivanov-prime) [inline src/Lykke.AlgoStore.Api/Controllers/AlgoController.cs]: should be renamed, no need for Metadata work anymore

Comment (iliyan-ivanov-prime) [reply src/Lykke.AlgoStore.Api/Controllers/AlgoController.cs]: Can be named GetAlgoInformation, both swagger name and method name should be renamed

File: src/Lykke.AlgoStore.Api/Controllers/AlgoController.cs [hunk 2]
```diff
@@ -59,5 +66,190 @@ public AlgoController(IAlgoStoreClientDataService clientDataService)
...
+        }
+
+        [HttpPost("metadata/cascadeDelete")]
+        [SwaggerOperation("CascadeDeleteAlgoMetadata")]
+        [ProducesResponseType((int)HttpStatusCode.NoContent)]
+        [ProducesResponseType(typeof(BaseErrorResponse), (int)HttpStatusCode.NotFound)]
+        [ProducesResponseType(typeof(BaseErrorResponse), (int)HttpStatusCode.InternalServerError)]
+        [ProducesResponseType(typeof(ErrorResponse), (int)HttpStatusCode.BadRequest)]
+        public async Task<IActionResult> DeleteAlgoMetadata([FromBody]ManageImageModel model)
+        {
```
Comment (iliyan-ivanov-prime) [inline src/Lykke.AlgoStore.Api/Controllers/AlgoController.cs]: Same comment as above

File: src/Lykke.AlgoStore.Core/Services/IAlgosService.cs [hunk 3]
```diff
@@ -5,26 +5,20 @@
...
         Task<PublicAlgoData> RemoveFromPublicAsync(PublicAlgoData data, string clientId);
 
-        Task<AlgoClientInstanceData> ValidateCascadeDeleteClientMetadataRequestAsync(ManageImageData data);
         Task SaveAlgoAsBinaryAsync(string clientId, UploadAlgoBinaryData dataModel);
         Task DeleteMetadataAsync(ManageImageData data);
         Task SaveAlgoAsStringAsync(string clientId, UploadAlgoStringData dataModel);
         Task<string> GetAlgoAsStringAsync(string clientId, string algoId);
-        Task<List<AlgoClientInstanceData>> GetAllAlgoInstanceDataByAlgoIdAndClientIdAsync(CSharp.AlgoTemplate.Models.Models.BaseAlgoData data);
 
         Task<List<AlgoClientInstanceData>> GetAllAlgoInstanceDataByClientIdAsync(string clientId);
```
Comment (iliyan-ivanov-prime) [inline src/Lykke.AlgoStore.Core/Services/IAlgosService.cs]: This is dealing with algo instances, should it be in this service ?

File: src/Lykke.AlgoStore.Core/Services/IAlgosService.cs [hunk 4]
```diff
@@ -5,26 +5,20 @@
...
+    public interface IAlgosService
     {
         Task<List<AlgoRatingMetaData>> GetAllAlgosWithRatingAsync();
 
         Task<PublicAlgoData> AddToPublicAsync(PublicAlgoData data, string clientId);
         Task<PublicAlgoData> RemoveFromPublicAsync(PublicAlgoData data, string clientId);
 
-        Task<AlgoClientInstanceData> ValidateCascadeDeleteClientMetadataRequestAsync(ManageImageData data);
         Task SaveAlgoAsBinaryAsync(string clientId, UploadAlgoBinaryData dataModel);
         Task DeleteMetadataAsync(ManageImageData data);
```
Comment (iliyan-ivanov-prime) [inline src/Lykke.AlgoStore.Core/Services/IAlgosService.cs]: Maybe rename this method to "DeleteAlgoAsync"

**Review by ghost — APPROVED** [review]

**Review by iliyan-ivanov-prime — APPROVED** [review]

**Review by mjoksa — APPROVED** [review]

**Pull request merged**